src/test/java/org/apache/commons/compress/archivers/tar/SparseFilesTest.java

```diff
@@ -119,9 +120,8 @@ private String getTarBinaryHelp() throws IOException {
     }
 
     @Test
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
:ok_man:

Reply:
Why would we disable a test that is passing?

Reply:
It's disabled on Windows. The old test had this line removed:

```java
assumeFalse("This test should be ignored on Windows", isOnWindows);
```

Reply:
As @kinow said those tests where already disabled by the assume statements in the tests. Would be nice if you can double check that I didn't mess up the logic 😉